src/astro/sql/operators/transform.py

```diff
@@ -0,0 +1,197 @@
+import inspect
+from typing import Callable, Dict, Iterable, Mapping, Optional, Union
+
+from airflow.decorators.base import DecoratedOperator, task_decorator_factory
+from sqlalchemy.sql.functions import Function
+
+from astro.databases import create_database
+from astro.sql.table import Table
+from astro.utils.dataframe_function_handler import DataframeFunctionHandler
+from astro.utils.sql_handler import SQLHandler
+from astro.utils.table_handler import TableHandler
+
+
+class TransformOperator(
+    SQLHandler, DataframeFunctionHandler, DecoratedOperator, TableHandler
+):
+    """Handles all decorator classes that can return a SQL function"""
+
+    # todo: Add docstrings
+    def __init__(
+        self,
+        conn_id: Optional[str] = None,
+        autocommit: bool = False,
```

Comment:
I can't see where this property is being used at the moment, should we remove it from all operators for now?